hmp/models/forward.py

```diff
@@ -0,0 +1,295 @@
+"""Models to estimate event probabilities."""
+
+import gc
+import itertools
+import multiprocessing as mp
+from itertools import cycle, product
+from warnings import resetwarnings, warn
+
```

Comment:
There's probably a bunch of librabies that we no longer need in the individual modules right?

Reply:
I removed most of the unused imports.